ls1intum/Artemis
# `General`: Remove instructor's course and exercise dashboard

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

[ ]* - not needed

### Motivation and Context


Instructor course and exercise dashboards don't add much new information in comparison with course and exercise assessment dashboards. This PR removes instructor course and exercise dashboards as well as all code related to it that is no longer needed.

### Description

Removed the components on the client side, server endpoints that were providing data for the component, tests, adjusted the translation strings as well as file imports that were inside the instructor dashboard folder, but are moved to the dashboard root folder.

### Steps for Testing

1. Log in to Artemis as instructor
2. Navigate to Course Administration
3. Instructor course dashboard button should not be there and Artemis should function as before without any issues.

### Review Progress





#### Code Review
#### Manual Tests
